CHANGELOG.md

```diff
@@ -28,8 +28,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     logs, likewise use a pattern like the ES\_DISTRIBUTION\_TARGET\_PATTERN.
 
 ### Changed
+- **CUMULUS-2542**
+  - Created a new ASYNC dropdown in order to be used to search Collections since some Collections were not showing
+  - Created a new file for styles to be imported in Simple-Dropdown and Async-Dropdown
```

Comment:
Can you move this so that the ticket numbers are in numerical order?

Reply:
Fixed